From: Balbir Singh <balbirs@nvidia.com>
Subject: mm-allow-compound-zone-device-pages-fix-fix
Date: Fri, 21 Feb 2025 12:27:49 +1100

 - Add {} around the if statement as per coding style
 - Add Reviewed-by/Acked-by tags

Link: https://lkml.kernel.org/r/20250219231337.364133-1-balbirs@nvidia.com
Link: https://lkml.kernel.org/r/20250221012749.506513-1-balbirs@nvidia.com
Signed-off-by: Balbir Singh <balbirs@nvidia.com>
Reviewed-by: Alistair Popple <apopple@nvidia.com>
Acked-by: David Hildenbrand <david@redhat.com>
Cc: Jason Gunthorpe <jgg@ziepe.ca>
Cc: Dan Williams <dan.j.williams@intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/migrate_device.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/migrate_device.c~mm-allow-compound-zone-device-pages-fix-fix
+++ a/mm/migrate_device.c
@@ -154,9 +154,9 @@ again:
 			}
 			page = vm_normal_page(migrate->vma, addr, pte);
 			if (page && !is_zone_device_page(page) &&
-			    !(migrate->flags & MIGRATE_VMA_SELECT_SYSTEM))
+			    !(migrate->flags & MIGRATE_VMA_SELECT_SYSTEM)) {
 				goto next;
-			else if (page && is_device_coherent_page(page)) {
+			} else if (page && is_device_coherent_page(page)) {
 				pgmap = page_pgmap(page);
 
 				if (!(migrate->flags &
_
